openapi/graphhopper.directions/openapi.yml

```diff
@@ -0,0 +1,4488 @@
+openapi: 3.0.0
+info:
+  title: GraphHopper Directions API
+  x-display:
+    label: GraphHopper Directions
+    iconPath: "resources/GraphHopperDirections.svg"
+  x-init-description: >- 
+    The connector initialization requires setting the API credentials.
+    Create an [GraphHopper account](https://support.graphhopper.com/a/solutions/articles/44001976025) and obtain tokens following 
+    [this guide](https://docs.graphhopper.com/#section/Authentication).
+  description: |
+    This is a generated connector for [GraphHopper Directions API v1.0.0](https://docs.graphhopper.com/) OpenAPI Specification. 
+    
+    With the [GraphHopper Directions API](https://www.graphhopper.com/products/) you can integrate A-to-B route planning, turn-by-turn navigation,
+    route optimization, isochrone calculations and other tools in your application.
```

Comment:
I suggest keeping this sentence only in here. It is nice to see that entire description generated as a client class doc comment. But content is too much.

Reply:
Reduced the content.